README.md

```diff
@@ -38,14 +38,18 @@ Dict{Symbol,Any} with 5 entries:
   :head        => :function
   :whereparams => Any[:T]
 
+
 julia> def[:name] = :g;
 
 julia> def[:head] = :(=);
 
-julia> def[:body] = :(x * y);
+julia> args_tuple_expr(def)
+:((x, y))
+
+julia> def[:body] = :(*($(args_tuple_expr(def))...));
```

Comment:
is this right?

Reply:
why wouldn't it be.
The next line looks right to me.